apps/beeai-web/package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "dev": "NODE_OPTIONS=\"--no-experimental-global-navigator\" next dev",
     "build": "NODE_OPTIONS=\"--no-experimental-global-navigator\" next build",
-    "start": "NODE_OPTIONS=\"--no-experimental-global-navigator\" next start",
+    "preview": "NODE_OPTIONS=\"--no-experimental-global-navigator\" next start",
```

Comment:
I would definitely stick to the name next uses and match it:
```
start: "... start",
```

The PR could be then named:
```
chore(web): add start task
```

Reply:
That is interesting! LD
I would personally not try to match it. One is Vite, the other one is Next.